src/storage/interface.js

```diff
@@ -30,6 +30,7 @@ class StorageInterface {
    */
   async clear (name) { throw new Error('storage clear method not implemented') }
   async refresh () { throw new Error('storage refresh method not implemented') }
+  async getTTL () { throw new Error('storage getTTL method not implemented') }
```

Comment:
This is out of scope and should not be there, since the Storage interface is not using this method